style.css

```diff
@@ -0,0 +1,101 @@
+@media only screen and (max-width: 600px) {
+  body {
+    background-color: black;
+    max-height: 1000px;
+  }
+
+  header {
+    background-color: black;
+  }
+
+  .home {
+    background-image: url("images/Group.png");
+    background-repeat: no-repeat;
+    background-attachment: fixed;
+    background-position: bottom right;
+  }
+
+  .nav {
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    margin-left: 0;
+    background-color: #3c3a39;
+    padding-top: 56px;
+  }
+
+  .logo {
+    font-family: "Roboto", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 18px;
+    line-height: 24px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    color: #fff;
+    text-decoration: none;
+    padding-left: 24px;
+    margin-bottom: 12px;
+  }
+
+  .humburger {
+    color: white;
+    padding-right: 36px;
+  }
+
+  .title {
+    width: 327px;
+    height: 124px;
+    font-family: "Crete Round", serif;
+    font-weight: 400;
+    font-size: 56px;
+    line-height: 64px;
+    margin: 120px 24px 24px;
+    letter-spacing: 0.37px;
+    color: #ff6b00;
+  }
+
+  .my-job {
+    width: 327px;
+    height: 27px;
+    font-family: "Roboto", sans-serif;
+    font-weight: 700;
+    font-size: 24px;
+    line-height: 32px;
+    display: flex;
+    align-items: center;
+    color: #fff5e1;
+    margin: 0 24px 24px;
+  }
+
+  .intro {
+    width: 327px;
+    height: 123px;
+    font-family: "Poppins", sans-serif;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    color: #fff;
+    margin: 12px 24px;
+  }
+
+  .list {
+    list-style-type: none;
+    display: flex;
+    width: 200px;
+    justify-content: space-between;
+  }
+
+  .social-item {
+    height: 20px;
+    width: 18px;
+    color: white;
+  }
```

Comment:
- [ ] Kindly check the size of your social icons, they are smaller. Following the size of your FIGMA design would be great to impress your future visitors 💯 

![image](https://user-images.githubusercontent.com/93630700/204954159-fff61c8e-17e0-44f2-bd68-544b48461b72.png)